offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -135,7 +122,7 @@ class QuotationDetails {
         htmlContent.getElementById(elementId).append(ItemPrintout.subTableFooter(productGroup))
 
         // Update footer Prices
-        addSubTotalPrices(productGroup, items)
+        addSubTotalPrices(productGroup, items, subTotal)
```

Comment:
method signature is not correct I could only find `addSubTotalPrices(ProductGroup, double)`

Reply:
thanks, I fixed it